### PR TITLE
Fixes #29280 - rpm content uploads

### PR DESCRIPTION
**Pull request by sjha4**
Needs more
- [x] Testing
- [x] Tests

### CODE REVIEW

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

File: app/lib/actions/pulp3/orchestration/repository/upload_content.rb [hunk 1]
```diff
@@ -3,14 +3,12 @@ module Pulp3
...
           def plan(repository, smart_proxy, file, unit_type_id)
             sequence do
               content_backend_service = SmartProxy.pulp_master.content_service(unit_type_id)
-              content_href = nil
-              if unit_type_id == ::Katello::FileUnit::CONTENT_TYPE
-                content_list = content_backend_service.content_api.list("digest": Digest::SHA256.hexdigest(File.read(file[:path])))
-                content_href = content_list&.results&.first&.pulp_href unless content_list&.results&.empty?
-              end
+              content_list = content_backend_service.content_api.list("sha256": Digest::SHA256.hexdigest(File.read(file[:path])))
+              content_href = content_list&.results&.first&.pulp_href unless content_list&.results&.empty?
```
[the remark this answers is not in the document]
Comment (sjha4) [reply app/lib/actions/pulp3/orchestration/repository/upload_content.rb]: Yep..updated..

**Comment by jlsherrill** [comment]
Overall looks good, doing some testing!

**Pull request merged**